test/backend_test.in.cpp

```diff
@@ -1257,17 +1261,17 @@ NGRAPH_TEST(${BACKEND_NAME}, lrn)
     backend->call_with_validate(handle, {result}, {a});
 
     vector<float> expected{0.f,
-                           0.05325444f,
-                           0.03402646f,
-                           0.01869806f,
-                           0.06805293f,
-                           0.03287071f,
-                           0.00509002f,
-                           0.00356153f,
-                           0.00174719f,
-                           0.0012555f,
-                           0.00322708f,
-                           0.00235574f};
+                           0.3015113f,
```

Comment:
Where did these numbers come from? How can I generate new numbers?

Reply:
TF script I wrote, which first reproduced old numbers. Tweaked parameters to generate numbers that were less susceptible to float precision issues (e.g. used alpha of 3, because ngraph definition of lrn means that alpha / size - previously this resulted in using 1/3), and included 7 float digits (in contrast to previous 6).
Could add TF script in comments.